generated/nidaqmx/nidaqmx.proto

```diff
@@ -2835,6 +2835,31 @@ enum WriteStringAttribute {
   WRITE_ATTRIBUTE_SYNC_UNLOCKED_CHANS = 12608;
 }
 
+enum TriggerUsageTypesAttribute {
+  TriggerUsageTypes_ADVANCE = 1;
+  TriggerUsageTypes_PAUSE = 2;
+  TriggerUsageTypes_REFERENCE = 4;
+  TriggerUsageTypes_START = 8;
+  TriggerUsageTypes_HANDSHAKE = 16;
+  TriggerUsageTypes_ARM_START = 32;
+}
```

Comment:
Problems with this:
- These new enums are not used by any request/response messages.
- This doesn't extend the enums that are used by functions like `GetPhysicalChanAttributeInt32`, such as `PhysicalChannelInt32AttributeValues` (note the "Values" suffix).
- The suffix "Attribute" is incorrect because this is not an attribute.
- The enum value names should be UPPER_SNAKE_CASE.

Updating `get_attribute_enums_by_type` will address all of these problems.